event_manager/lib/validate_phone_number.rb

```
@@ -1,12 +1,18 @@
 class ValidatePhoneNumber
   def call(number)
-    return "Invalid phone format" unless number.kind_of?(String)
+    return 'Invalid phone format' unless number.is_a?(String)
```

I believe in this approach it we would be better to raise an exception instead